src/protobuf-net.Core/protobuf-net.Core.csproj

```diff
@@ -11,7 +11,11 @@
     <Description>Provides simple access to fast and efficient "Protocol Buffers" serialization from .NET applications</Description>
     <!--<DefineConstants>$(DefineConstants);TRACK_USAGE</DefineConstants>-->
   </PropertyGroup>
-  
+
+  <ItemGroup Condition="'$(TargetFramework)' == 'net462'">
+	<PackageReference Include="System.ValueTuple" Version="4.4.0" />
+  </ItemGroup>
```

Comment:
FWIW, I would advise against adding this. System.ValueTuple creates a lot of friction on full framework with bindings and redirects, so much that we avoid it completely whenever possible. Given there's not a lot of usage here, it'd make life less painful for users if this wasn't a dependency.

Reply:
@DeagleGross Nick is 100% right about this; friends to let friends use ValueTuple when there's any non-zero chance that you'll be on netfx - either explicitly (in this case) or indirectly (via netstandard)

Reply:
Hey, thanks for pointing that out - did not know that - will remove that!

Reply:
There's a bunch of 5 (Nick covers them here: https://nickcraver.com/blog/2020/02/11/binding-redirects/) where the runtime loader is just broken; they can't be used reliably on netfx

If you're an application developer, they're fine - you can just tweak the binding redirects once, job done

But if you're a library developer; it causes pain for *every* consumer; some % will know how to fix it, some % will give up, and some % will log support tickets

Reply:
thanks for explaining (great article btw), removed ValueTuple nuget package